src/Xamarin.Android.Build.Tasks/Utilities/RtxtParser.cs

```diff
@@ -63,6 +64,7 @@ public class RtxtParser {
 		static readonly char[] EmptyChar = new char [] { ' ' };
 		static readonly char[] CurlyBracketsChar = new char [] { '{', '}' };
 		static readonly char[] CommaChar = new char [] { ',' };
+		static readonly Regex ValidChars = new Regex (@"([^a-f0-9x, \{\}])+", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
At some point, should we start using the faster Regex source generator?

https://learn.microsoft.com/dotnet/standard/base-types/regular-expression-source-generators

Reply:
Probably. We can do that in one pass I think. Maybe not in this PR though :D